vulnerabilities/data_source.py

```diff
@@ -75,7 +75,8 @@ class InvalidConfigurationError(Exception):
 
 @dataclasses.dataclass
 class DataSourceConfiguration:
-    batch_size: int
+    # batch_size: int
```

Comment:
please remove the line instead of commenting it out

Reply:
Done